src/components/Square.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+const Square = ({
+  i,
+  squares,
+  setSquares,
+  remainingAlphabet,
+  setRemainingAlphabet,
+  errorMessage,
+  setErrorMessage,
+  maxNumberConsonants,
+}) => {
+  const vowels = ['A', 'E', 'I', 'O', 'U'];
+  const totalNumberOfConsonants = 27;
+  
+  const editInput = e => {
+    const newSquares = JSON.parse(JSON.stringify(squares));
+    const workRemainingAlphabet = JSON.parse(JSON.stringify(remainingAlphabet));
+    let newLetter = e.target.value.replace(/[^a-z]/gi, '').toUpperCase();
+    
+    // Check to see if you have reached the extent of your letter useage
+    let workErrorMessage = '';
+    if (
+      maxNumberConsonants < totalNumberOfConsonants - remainingAlphabet.length + 1 &&
+      vowels.indexOf(newLetter) === -1
```

Comment:
I still think this `vowels.indexOf(newLetter) === -1` occurs enough that you might as well make a util so its a least a little more readable

Reply:
Create function notVowel(letter)